Tests/SkipFoundationTests/Foundation/LocaleTests.swift

```diff
@@ -727,6 +727,197 @@ final class LocaleTests: XCTestCase {
+        "DOP": "Dominican Peso",
+        "DZD": "Algerian Dinar",
+        "EGP": "Egyptian Pound",
+        "ERN": "Eritrean Nakfa",
+        "ETB": "Ethiopian Birr",
+        "EUR": "Euro",
+        "FJD": "Fijian Dollar",
+        "FKP": "Falkland Islands Pound",
+        "GBP": "British Pound",
+        "GEL": "Georgian Lari",
+        "GHS": "Ghanaian Cedi",
+        "GIP": "Gibraltar Pound",
+        "GMD": "Gambian Dalasi",
+        "GNF": "Guinean Franc",
+        "GTQ": "Guatemalan Quetzal",
+        "GYD": "Guyanaese Dollar",
+        "HKD": "Hong Kong Dollar",
+        "HNL": "Honduran Lempira",
+        "HRK": "Croatian Kuna",
+        "HTG": "Haitian Gourde",
+        "HUF": "Hungarian Forint",
+        "IDR": "Indonesian Rupiah",
+        "ILS": "Israeli New Shekel",
+        "INR": "Indian Rupee",
+        "IQD": "Iraqi Dinar",
+        "IRR": "Iranian Rial",
+        "ISK": "Icelandic Króna",
+        "JMD": "Jamaican Dollar",
+        "JOD": "Jordanian Dinar",
+        "JPY": "Japanese Yen",
+        "KES": "Kenyan Shilling",
+        "KGS": "Kyrgystani Som",
+        "KHR": "Cambodian Riel",
+        "KMF": "Comorian Franc",
+        "KPW": "North Korean Won",
+        "KRW": "South Korean Won",
+        "KWD": "Kuwaiti Dinar",
+        "KYD": "Cayman Islands Dollar",
+        "KZT": "Kazakhstani Tenge",
+        "LAK": "Laotian Kip",
+        "LBP": "Lebanese Pound",
+        "LKR": "Sri Lankan Rupee",
+        "LRD": "Liberian Dollar",
+        "LSL": "Lesotho Loti",
+        "LYD": "Libyan Dinar",
+        "MAD": "Moroccan Dirham",
+        "MDL": "Moldovan Leu",
+        "MGA": "Malagasy Ariary",
+        "MKD": "Macedonian Denar",
+        "MMK": "Myanmar Kyat",
+        "MNT": "Mongolian Tugrik",
+        "MOP": "Macanese Pataca",
+        "MRU": "Mauritanian Ouguiya",
+        "MUR": "Mauritian Rupee",
+        "MVR": "Maldivian Rufiyaa",
+        "MWK": "Malawian Kwacha",
+        "MXN": "Mexican Peso",
+        "MYR": "Malaysian Ringgit",
+        "MZN": "Mozambican Metical",
+        "NAD": "Namibian Dollar",
+        "NGN": "Nigerian Naira",
+        "NIO": "Nicaraguan Córdoba",
+        "NOK": "Norwegian Krone",
+        "NPR": "Nepalese Rupee",
+        "NZD": "New Zealand Dollar",
+        "OMR": "Omani Rial",
+        "PAB": "Panamanian Balboa",
+        "PEN": "Peruvian Sol",
+        "PGK": "Papua New Guinean Kina",
+        "PHP": "Philippine Peso",
+        "PKR": "Pakistani Rupee",
+        "PLN": "Polish Zloty",
+        "PYG": "Paraguayan Guarani",
+        "QAR": "Qatari Riyal",
+        "RON": "Romanian Leu",
+        "RSD": "Serbian Dinar",
+        "RUB": "Russian Ruble",
+        "RWF": "Rwandan Franc",
+        "SAR": "Saudi Riyal",
+        "SBD": "Solomon Islands Dollar",
+        "SCR": "Seychellois Rupee",
+        "SDG": "Sudanese Pound",
+        "SEK": "Swedish Krona",
+        "SGD": "Singapore Dollar",
+        "SHP": "St. Helena Pound",
+        "SLE": "Sierra Leonean Leone",
```

Comment:
It looks like this is triggering the CI test case failure on the macOS side for `testCommonCurrencies`. I'm guessing because the OS it is running on (macos-13) doesn't consider "SLE" as one of the common currencies, but the version you are running locally, does. I recommend just making the test case a bit "fuzzier", maybe just checking that the `Locale.commonISOCurrencyCodes` is a subset of your hardcoded test list.

Reply:
Yeah, you're right. I think the en-US localized strings also differ across OS versions, so I'm going to remove them.